man/man5/zfs-module-parameters.5

```diff
@@ -686,6 +710,31 @@ Min time before an active prefetch stream can be reclaimed
 Default value: \fB2\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_abd_scatter_enabled\fR (int)
+.ad
+.RS 12n
+Disables ARC from using scatter/gather lists and forces all allocations to be
```

Comment:
I think it would be better like:
"Enables ARC to use scatter/gather lists and allows allocations to be non-linear in kernel memory."